cluster-autoscaler/processors/processors.go

```diff
@@ -78,7 +78,7 @@ func DefaultProcessors(options config.AutoscalingOptions) *AutoscalingProcessors
 	return &AutoscalingProcessors{
 		PodListProcessor:       pods.NewDefaultPodListProcessor(),
 		NodeGroupListProcessor: nodegroups.NewDefaultNodeGroupListProcessor(),
-		BinpackingLimiter:      binpacking.NewDefaultBinpackingLimiter(),
```

Comment:
It is a bit confusing that the default `BinpackingLimiter` is not the one returned by `NewDefaultBinpackingLimiter()`. Let's just remove `NewDefaultBinpackingLimiter()` now?

Reply:
Deprecated NoOpBinpackingLimiter